package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "optic-custom-rules",
+  "name": "name-of-custom-rules-package",
   "version": "1.0.0",
   "scripts": {
-    "publish": "OPTIC_TOKEN=$(grep OPTIC_TOKEN .env | cut -d '=' -f2) optic ruleset publish build/main.js",
```

Comment:
Should we go ahead and plan on renaming `ruleset publish` to `ruleset upload` just to keep this aligned?

Reply:
Yeah I'll put up a PR for that
=

Reply:
https://github.com/opticdev/optic/pull/1402